src/include/bootloader/grub/options.rb

```diff
@@ -71,14 +72,22 @@ def passwd_content
           Id(:use_pas),
           _("Prot&ect Boot Loader with Password"),
           true,
-          HBox(
-            HSpacing(2),
-            # text entry
-            Password(Id(:pw1), Opt(:hstretch), _("&Password")),
-            # text entry
-            HSpacing(2),
-            Password(Id(:pw2), Opt(:hstretch), _("Re&type Password")),
-            HStretch()
+          VBox(
+            HBox(
+              HSpacing(2),
+              # TRANSLATORS: checkbox entry
+              CheckBox(Id(:unrestricted_pw), _("A&llow to boot locked default entry without password")),
```

Comment:
Why are we talking about a "locked" default entry? I thought being locked is a global property for all entries together, controlled by the "Password" checkbox.